plugins/generator-1.19.4/forge-1.19.4/templates/livingentity/livingentity.java.ftl

```diff
@@ -545,16 +545,21 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
 	}
     </#if>
 
-	<#if hasProcedure(data.onMobTickUpdate)>
+	<#if hasProcedure(data.onMobTickUpdate) || hasProcedure(data.boundingBoxScale)>

```

Comment:
This should check for fixed value too (thanks to @Spectrall368 for discovering this bug).

Reply:
I don't fully understand the bug or the solution.

I see little sense in having to refresh dimensions every tick for a fixed scale value

Reply:
You mean it should be added to where initial spawn procedure call is generated?

Reply:
Maybe, I will check